### DISCUSSION TITLE
Can't switch the chain

### DISCUSSION THREAD

**Discussion by lishengling-1998** [WalletConnect/walletconnect-monorepo#2956] [Developer Support]
<img width="711" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/59638957/cfd271c8-de55-432a-ae6d-b39ecad219bb">

**Comment by lishengling-1998** [comment]
@bkrem

**Comment by crypblizz8** [comment]
Can you provide more reproducible code and context for this error message. Thanks
